static/js/ducks/source.js

```diff
@@ -277,6 +280,9 @@ export const fetchVizier = (sourceID) =>
 export const fetchPhotoz = (sourceID) =>
   API.POST(`/api/sources/${sourceID}/annotations/datalab`, FETCH_PHOTOZ);
 
+export const fetchAssociatedGCNs = (sourceID) =>
+  API.GET(`/api/source_in_gcns/${sourceID}`, FETCH_ASSOCIATED_GCNS);
```

Comment:
As discussed above, I don't really like `source_in_gcns`. But if you have good reasons to have this endpoint and the associated logic, a much better name could be `associated_gcns` or something like that.

Reply:
No good reason! As mentionned in the PR's description I want to change the names, they are definitely not clear enough. Thanks a lot for the suggestion!